src/sentry/types/actor.py

```diff
@@ -259,6 +259,18 @@ def is_user(self) -> bool:
         return self.actor_type == ActorType.USER
 
 
+class ActorOwned(Protocol):
```

Comment:
How do we apply this protocol to the classes?

Reply:
Protocols don't have to be applied to classes. They can be used as type annotation in methods/returns and match with [structural subtyping](https://typing.readthedocs.io/en/latest/spec/protocol.html#protocols). There aren't any references to the protocol yet, but I included it in case we need it, and to document the interactions that `Actor` can have with ORM models.